tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -48,14 +51,39 @@ void ROOT::RDF::RunGraphs(std::vector<RResultHandle> handles)
    std::set<RResultHandle, decltype(sameGraph)> s(handles.begin(), handles.end(), sameGraph);
    std::vector<RResultHandle> uniqueLoops(s.begin(), s.end());
 
+   // Trigger jitting. One call is enough to jit the code required by all computation graphs.
+   TStopwatch sw;
+   sw.Start();
+   {
+      auto silenceRDFLogs = ROOT::Experimental::RLogScopedVerbosity(ROOT::Detail::RDF::RDFLogChannel(),
```

Comment:
I would add a comment that this is done to avoid printing also the logging from RLoopManager::Jit

Reply:
done, but wasn't it clear from the code?

https://github.com/root-project/root/blob/0f333ad41d690735c1cd8d201ac3b68448ae547b/tree/dataframe/src/RDFHelpers.cxx#L57-L61

Reply:
Yes probably it was, I just prefer to be more verbose sorry :smile: